SECP-service/src/main/java/com/visucius/secp/DTO/GroupCreationRequest.java

```diff
@@ -0,0 +1,27 @@
+package com.visucius.secp.DTO;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Set;
+
+public class GroupCreationRequest {
+
+    @JsonProperty
+    public String name;
+    @JsonProperty
+    public Set<Long> users;
```

Comment:
Group creation request should not have list of users. The server should be able to figure out the users from the roles provided.